pkg/bloomgateway/util.go

```diff
@@ -28,23 +28,17 @@ func getFromThrough(refs []*logproto.ShortRef) (model.Time, model.Time) {
 	}
 
 	maxItem := slices.MaxFunc(refs, func(a, b *logproto.ShortRef) int {
-		if a.Through > b.Through {
-			return 1
-		} else if a.Through < b.Through {
-			return -1
-		}
-		return 0
+		return int(a.Through) - int(b.Through)
```

Comment:
nit: `model.Time` is int64
```suggestion
		return int64(a.Through) - int64(b.Through)
```